### PR TITLE
Seasalt

### PR DESCRIPTION
**Pull request by dulte**
Sea salt pm 25, and more

### CODE REVIEW

**Comment by jgliss** [comment]
@dulte can you check the failing tests first and make sure CI passes?

**Comment by dulte** [comment]
It was the test function that wasn't updated with the new definition of sea salt. This is fixed now

**Comment by jgliss** [comment]
@dulte we just merged  #410 into main-dev, could you add the updates to this branch before we merge?

**Comment by dulte** [comment]
Pulled the newest updates from main-dev and pushed it to this branch

File: pyaerocom/data/emep_variables.ini
Comment (jgliss) [inline pyaerocom/data/emep_variables.ini:72]: I guess this is still concsspm25 in ebas_config.ini. Also you probably need to add the new variable concssf in variables.ini.

File: pyaerocom/io/read_mscw_ctm.py
Comment (jgliss) [inline pyaerocom/io/read_mscw_ctm.py:144]: Ah, then co-location works if you use concsspm25!

**Review by jgliss — APPROVED** [review]
Looks good (but you might need to add concssf in variables.ini, not sure if it exists)

**Review by jgliss — CHANGES_REQUESTED** [review]
Actually, cleanup of glob_defaults needed before approval, sorry for the confusion.

**Comment by dulte** [comment]
I think most of the duplicates from glob default should be gone now

**Review by jgliss — COMMENTED** [review]
Perhaps you could add a test module for glob_defaults in tests/aeroval and add a test to make sure there no duplicates. Not important to do that now, but I think this should be tested before we merge.

**Comment by dulte** [comment]
Dicts made with {} will only keep the last duplicate entry, so that as soon as var_web_info actually becomes a dict in python, there will be no duplicates, meaning that we can test for it. If we change {} to dict(), it will crash instead, which might be better

**Comment by jgliss** [comment]
> Dicts made with {} will only keep the last duplicate entry, so that as soon as var_web_info actually becomes a dict in python, there will be no duplicates, meaning that we can test for it. If we change {} to dict(), it will crash instead, which might be better

Sounds good!

**Review by jgliss — APPROVED** [review]
Looks good

**Pull request merged**
